packages/flutter/test/material/tab_bar_theme_test.dart

```diff
@@ -20,9 +20,9 @@ const List<Tab> _tabs = <Tab>[
   Tab(text: _tab3Text, icon: Icon(Icons.looks_3)),
 ];
 
-Widget _buildTabBar({ List<Tab> tabs = _tabs }) {
-  final TabController _tabController = TabController(length: 3, vsync: const TestVSync());
+final TabController _tabController = TabController(length: 3, vsync: const TestVSync());
```

Comment:
This change is subtly different. Now all of the tab test share the same tab controller which reduces isolation. I don't think that's what we want...

Reply:
Good point, changed this to a getter, that way every caller gets a new instance of `TabController`